static/js/containers/FinancialAidCalculator.js

```diff
@@ -56,7 +55,7 @@ const currencySelect = (update, current) => (
 
 const salaryUpdate = R.curry((update, current, e) => {
   let newEdit = R.clone(current);
-  newEdit.income = sanitizeNumberString(20, e.target.value);
+  newEdit.income = e.target.value
```

Comment:
this wont allow user to enter (.) i.e 2000.00 and see error message.
it changes 2000.00 to 200000. try ur self, copy 2000.00  and paste in field you will see it change into 200000